pkg/cache/internal/informers.go

```diff
@@ -49,14 +49,19 @@ type InformersOpts struct {
 	Selector              Selector
 	Transform             cache.TransformFunc
 	UnsafeDisableDeepCopy bool
+	WatchErrorHandler     cache.WatchErrorHandler
```

Comment:
Question is how could users set this flied in InformersOpts?

Reply:
The user can set this the same way we allow them to pass in transform in the InformersOpts. They’d pass the function in and it’ll be set when the mapping function sets the `WatchErrorHandler` using that method after the SharedIndexInformer is created.

Reply:
Very good point.  I can put this in the `Options` struct and allow this to be overridden within the instantiation of the cache.   Being that this only has one, it doesn't need to be done `byObject` so I don't have to put it there.  The resultant handler will be the one passed in or the `toolscache.DefaultWatchErrorHandler`.  I will make the changes. 

```
// There's only one handler, so if you call this multiple times, last one
// wins; calling after the informer has been started returns an error.
```